hca/util/__init__.py

```diff
@@ -271,6 +273,11 @@ def __init__(self, config=None, swagger_url=None, **session_kwargs):
             for http_method, method_data in path_data.items():
                 self._build_client_method(http_method, http_path, method_data)
 
+    @staticmethod
+    def _audience(self):
+        return _deep_get(self.swagger_spec,["securityDefinitions","OauthSecurity","x-audience"]) or \
```

Comment:
Out of curiosity, is there a reason that `_deep_get` is used instead of addressing the key directly? It looks like `_deep_get` will still throw a `KeyError` if the key in question doesn't exist.

Reply:
i updated it: all credit here https://stackoverflow.com/a/36131992